c7n/resources/kafka.py

```diff
@@ -32,6 +32,15 @@ class resource_type(TypeInfo):
         filter_name = 'ClusterNameFilter'
         filter_type = 'scalar'
 
+    def augment(self, resources):
+        for r in resources:
+            if 'Tags' not in r:
```

Comment:
hmm.. we shouldn't just continue here, we should append to results and continue

Reply:
its actually okay, we're just returning the resources, not filtering to a separate list.